AutomatorServer/ios/AutomatorServerUITests/PatrolServer.swift

```diff
@@ -34,6 +34,10 @@ struct Selector: Codable {
   var text: String
 }
 
+struct DarkModeCommand: Codable {
+  var appId: String
```

Comment:
Since in dart part you call this `bundleId` I'd prefer to keep the naming consistent

Reply:
This command also exists on Android and there we'd have to name it `packageName`, but this would require us to make different requests for every platform with no real benefit. IGL it's okay.

Anyway this won't be a problem once we migrate the communication to protobufs and gRPC in #338.